### PR TITLE
chore: move to standalone

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

File: projects/cdk/directives/control/control.directive.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import {NgControl} from '@angular/forms';
 export class TuiControlDirective {
     private readonly ngControl = inject(NgControl);
 
-    protected get control(): AbstractControl {
+    public get control(): AbstractControl {
```
Comment (vladimirpotekhin) [inline projects/cdk/directives/control/control.directive.ts:13]: why public ?

Comment (splincode) [reply projects/cdk/directives/control/control.directive.ts:13]: That is, most likely it was assumed that this directive could be injected and access the control

File: projects/kit/components/accordion/test/accordion.component.spec.ts [hunk 2]
```diff
@@ -40,6 +39,14 @@ describe('Accordion', () => {
     let testComponent: TestComponent;
 
     @Component({
+        standalone: true,
+        imports: [
+            TuiAccordion,
+            ReactiveFormsModule,
+            TuiTextfieldControllerModule,
```
Comment (nsbarsukov) [inline projects/kit/components/accordion/test/accordion.component.spec.ts:46]: Could it be dropped ?

File: projects/kit/components/accordion/test/accordion.component.spec.ts [hunk 3]
```diff
@@ -40,6 +39,14 @@ describe('Accordion', () => {
...
+            TuiDataList,
+            TuiDataListWrapper,
```
Comment (nsbarsukov) [inline projects/kit/components/accordion/test/accordion.component.spec.ts:48]: Could we drop them to ?

**Review by nsbarsukov — APPROVED** [review]

File: projects/kit/components/accordion/test/accordion.component.spec.ts (hunk 3 above)
Comment (splincode) [reply projects/kit/components/accordion/test/accordion.component.spec.ts]: fixed

File: projects/kit/components/accordion/test/accordion.component.spec.ts (hunk 2 above)
Comment (splincode) [reply projects/kit/components/accordion/test/accordion.component.spec.ts]: fixed

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**